<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateDetailsDialog</class>
 <widget class="QDialog" name="DateDetailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Date details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbDate">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dteDate"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbHasYear">
     <property name="text">
      <string>Set year for date</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbHasTime">
     <property name="text">
      <string>Set time for date</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbHasTimeZone">
     <property name="text">
      <string>Set time zone</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="sbHour">
       <property name="minimum">
        <number>-11</number>
       </property>
       <property name="maximum">
        <number>11</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbHour">
       <property name="text">
        <string>h</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbMin">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbMin">
       <property name="text">
        <string>m</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DateDetailsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DateDetailsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
